Gems/OpenXRVk/Code/Source/OpenXRVkSwapChain.cpp

```diff
@@ -158,10 +160,20 @@ namespace OpenXRVk
                 AZ_Printf("OpenXRVk", "Swapchain Formats: %s\n", swapchainFormatsString.c_str());
             }
 
+            double xrViewResolutionScale = 1.0;
+            if (AZ::SettingsRegistryInterface* settingsRegistry = AZ::SettingsRegistry::Get())
```

Comment:
Can add a check on the `settingsRegistry` pointer...

`if (auto settingsRegistry = AZ::SettingsRegistry::Get(); settingsRegistry != nullptr)`

Reply:
It's not necessary, I've seen both ways in the code base. But I'm ok changing it.